tests/centos9-build/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM quay.io/centos/centos:stream8
```

Comment:
Should this be stream9?

Reply:
BTW should we setup a shortname for centos to point at quay.io/centos/centos ?

Reply:
yes, thanks for spotting it!

Fixed and pushed a new version

Reply:
I've checked and shortnames already uses `quay.io/centos/centos`

Reply:
Nice.